<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Transaction</class>
 <widget class="QDialog" name="Transaction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(219, 255, 252);</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_card">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>80</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_password">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>350</y>
     <width>126</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Second password.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>63</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb();</string>
   </property>
   <property name="text">
    <string>Card:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_cvv2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>150</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_year">
   <property name="geometry">
    <rect>
     <x>674</x>
     <y>227</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_month">
   <property name="geometry">
    <rect>
     <x>838</x>
     <y>230</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pssword">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>354</y>
     <width>261</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_robot">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>280</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>I am not a robot</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>430</y>
     <width>98</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/cancel for kala.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_payment">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>430</y>
     <width>100</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Patment.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>150</y>
     <width>63</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb();</string>
   </property>
   <property name="text">
    <string>CVV2:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>225</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb();</string>
   </property>
   <property name="text">
    <string>Month:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>613</x>
     <y>226</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb();</string>
   </property>
   <property name="text">
    <string>Year:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>220</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb();</string>
   </property>
   <property name="text">
    <string>Expiration date:</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>391</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb();
color: rgb(198, 255, 250);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb();
color: rgb(198, 255, 250);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>123</x>
      <y>60</y>
      <width>20</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb();
color: rgb(198, 255, 250);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_seller">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>Seller information</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb();</string>
    </property>
    <property name="text">
     <string>Phone:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb();</string>
    </property>
    <property name="text">
     <string>Price:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb();</string>
    </property>
    <property name="text">
     <string>Alireza</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_phone">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb();</string>
    </property>
    <property name="text">
     <string>09140409531</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_price">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 0);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb();</string>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1011</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/internet.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
